config/filter.d/ignorecommands/apache-fakegooglebot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Based on: https://isc.sans.edu/forums/diary/When+Google+isnt+Google/15968/
```

Comment:
It's interesting to make it more generic, I'm trying to think of a use-case where similar matching makes sense (smtp comes to mind, but that's built in and configurable mostly within the MTA).

I'm inclined to suggest, if someone uses the same logic of rDNS name matching in the future, we should abstract them to reduce duplicate code in ignorecommands scripts. I'm afraid creating a new argument name (and implementing its appropriate getters/setters/testcases) with only one implementation creates more sprawl than it hopes to prevent. Just my two cents thinking about it initially.